prow/plugins.yaml

```diff
@@ -72,101 +69,117 @@ approve:
   implicit_self_approve: true
 
 plugins:
-  istio/istio:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
-
   istio/api:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
+    - needs-rebase
+    - size
+    - trigger
+
+  istio/bots:
+    - needs-rebase
+    - size
+    - trigger
+
+  istio/cni:
+    - needs-rebase
+    - size
+    - trigger
+
+  istio/common-files:
+    - needs-rebase
+    - size
+    - trigger
 
   istio/installer:
```

Comment:
no wip? Seems like something we want

Reply:
In  general, I don't want to configure any of the /foo commands. You can set the WIP progress label through the GH UI instead. This eliminate a spurious email message to all reviewers and keeps the PR cleaner.

Reply:
the wip one also applied it based on the pr being a draft or having wip in the title. if we change this we should send out a notice to people so they don't think their PR will be marked as wip and have it get merged

Reply:
Oh, I didn't know about that behavior. I've added wip then.

Reply:
Please remember to configure mergify to stay aware of these labels.